labgrid/driver/usbloader.py

```diff
@@ -286,3 +286,54 @@ def load(self, filename=None, phase=None):
     @step()
     def execute(self):
         self.run_tool('exe', '%#x' % self.loadaddr)
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class TegraUSBDriver(Driver, BootstrapProtocol):
+    bindings = {
+        'loader': {'TegraUSBLoader', 'NetworkTegraUSBLoader'},
+    }
+
+    loadaddr = attr.ib(validator=attr.validators.instance_of(int))
+    bct = attr.ib(validator=attr.validators.instance_of(str))
+    usb_path = attr.ib(validator=attr.validators.instance_of(str))
+    image = attr.ib(default=None)
+
+    def __attrs_post_init__(self):
+        super().__attrs_post_init__()
+        # FIXME make sure we always have an environment or config
+        if self.target.env:
+            self.tool = self.target.env.config.get_tool('tegrarcm')
+        else:
+            self.tool = 'tegrarcm'
+
+    def on_activate(self):
+        pass
+
+    def on_deactivate(self):
+        pass
+
+    @Driver.check_active
+    @step(args=['filename'])
+    def load(self, filename=None, phase=None):
+        if filename is None and self.image is not None:
+            filename = self.target.env.config.get_image_path(self.image)
+        mf = ManagedFile(filename, self.loader)
+        mf.sync_to_resource()
+
+        pathname = mf.get_remote_path()
+        args = [self.tool, '--bct=' + self.bct,
```

Comment:
Doesn't the bct need to be pushed to the exporter as well (using `ManagedFile`)?